test/check-machines-storage-pools

```diff
@@ -739,6 +734,69 @@ class TestMachinesStoragePools(VirtualMachinesCase):
         b.click(".pf-v5-c-card .pf-v5-c-card__header button:contains(Storage pool)")
         b.wait_visible("#storage-pools-listing")
 
+        # Check for deleting a pool whose disk is used by a VM
+        b.click("#pool-images-system-action-kebab button")
+        b.click("#delete-pool-images-system")
+        b.set_checked("#storage-pool-delete-volumes", True)
+        b.wait_in_text(".pf-v5-c-backdrop .pf-v5-c-helper-text__item-text", "Pool's volumes are used by")
+        b.wait_visible(".pf-v5-c-backdrop footer button[aria-disabled=true]:contains(Delete)")
+
+        # Check "Cancel" button
+        b.click(".pf-v5-c-backdrop footer button:contains(Cancel)")
+        b.wait_not_present(".pf-v5-c-backdrop")
+
+        # Check "X" button of the dialog
+        b.click("#pool-images-system-action-kebab button")
+        b.click("#delete-pool-images-system")
+        b.click(".pf-v5-c-backdrop button[aria-label=Close]")
+        b.wait_not_present(".pf-v5-c-backdrop")
+
+        # Check error shown beside the title, not in the footer
+        # Delete the VM, or the deletion button will be disabled
+        m.execute("virsh dumpxml subVmTest1 > /tmp/subVmTest1.xml")
+        m.execute("virsh undefine subVmTest1")
+        m.execute(f'chattr +a {args["image"]}')
+        self.addCleanup(m.execute, f'chattr -a {args["image"]}')
+
+        b.click("#pool-images-system-action-kebab button")
+        b.click("#delete-pool-images-system")
+        b.set_checked("#storage-pool-delete-volumes", True)
+        b.click(".pf-v5-c-backdrop footer button[aria-disabled=false]:contains(Delete)")
```

Comment:
This is failing our test farm at https://artifacts.dev.testing-farm.io/85d99f8d-87ee-4d18-89af-2ee7fc1261e7/

Reply:
But that should be some flake, because I don't see how this would be failing our tests